src/locales/fr/WCAG.json

```
@@ -0,0 +1,729 @@
+                },
+                "4": {
+                    "DESCRIPTION": "L’espacement entre les lignes (interlignage) est d’une valeur d’au moins 1,5 dans les paragraphes et l’espacement entre les paragraphes est au moins 1,5 fois plus grand que la valeur de l’interligne."
+                },
+                "5": {
+                    "DESCRIPTION": "La taille du texte peut être redimensionnée jusqu’à 200 pour cent sans l’aide d’une technologie d’assistance et sans que l’utilisateur soit obligé de faire défiler le texte horizontalement pour lire une ligne complète dans une fenêtre plein écran."
+                }
+            }
+        },
+        "1.4.9": {
+            "TITLE": "Texte sous forme d’image (sans exception)",
+            "DESCRIPTION": "Le texte sous forme d’image est utilisé seulement pour du texte purement décoratif ou lorsqu’une présentation spécifique du texte est essentielle à l’information véhiculée.",
+            "DETAILS": {}
+        },
+        "1.4.10": {
+            "TITLE": "Redistribution",
+            "DESCRIPTION": "Le contenu peut être présenté sans perte d’information ou de fonctionnalité et sans nécessité de défilement dans les deux dimensions pour :",
+            "DETAILS": {
+                "1": {
+                    "DESCRIPTION": "Un contenu à défilement vertical avec une largeur équivalente à 320 pixels CSS ;"
+                },
+                "2": {
+                    "DESCRIPTION": "Un contenu à défilement horizontal avec une hauteur équivalente à 256 pixels CSS."
+                }
+            }
+        },
+        "1.4.11": {
+            "TITLE": "Contraste du contenu non textuel",
+            "DESCRIPTION": "La présentation visuelle des éléments suivants a un rapport de contraste d’au moins 3:1 avec la ou les couleurs adjacentes :",
+            "DETAILS": {
+                "1": {
+                    "DESCRIPTION": "Informations visuelles nécessaires à l’identification des composants et des états de l’interface utilisateur, à l’exception des composants inactifs ou lorsque l’apparence du composant est déterminée par l’agent utilisateur et non modifiée par l’auteur.",
+                    "TITLE": "Composants d’interface utilisateur"
+                },
+                "2": {
+                    "DESCRIPTION": "Parties d’éléments graphiques nécessaires à la compréhension du contenu, sauf si une présentation spécifique de ces éléments est essentielle à l’information transmise.",
+                    "TITLE": "Objets graphiques"
+                }
+            }
+        },
+        "1.4.12": {
+            "TITLE": "Espacement du texte",
+            "DESCRIPTION": "Dans un contenu mis en œuvre via un langage de balisage qui prend en charge les propriétés de style de texte suivantes, il n’y a aucune perte de contenu ou de fonctionnalité lorsqu’on applique toutes les valeurs ci-dessous sans modifier aucune autre propriété de style :",
```

```suggestion
            "DESCRIPTION": "Dans un contenu implémenté via un langage de balisage qui prend en charge les propriétés de style de texte suivantes, il n’y a aucune perte de contenu ou de fonctionnalité lorsqu’on applique toutes les valeurs ci-dessous sans modifier aucune autre propriété de style :",
```

See https://www.w3.org/Translations/WCAG21-fr/#text-spacing
Please use the exact wording of the Authorized Translation.